src/main/java/org/dependencytrack/resources/v1/TeamResource.java

```diff
@@ -220,6 +225,32 @@ public Response deleteTeam(Team jsonTeam) {
         }
     }
 
+    @GET
+    @Path("/visible")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Returns a list of Teams that are visible", description = "<p></p>")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "The Visible Teams", content = @Content(schema = @Schema(implementation = VisibleTeams.class))),
+            @ApiResponse(responseCode = "401", description = "Unauthorized")
+    })
+    public Response availableTeams() {
+        try (QueryManager qm = new QueryManager()) {
+            Principal user = getPrincipal();
+            List<Team> userTeams = new ArrayList<Team>();
+            if (user instanceof final UserPrincipal userPrincipal) {
+                userTeams = userPrincipal.getTeams();
+            } else if (user instanceof final ApiKey apiKey) {
+                userTeams = apiKey.getTeams();
+            }
+            boolean required = qm.isEnabled(ConfigPropertyConstants.ACCESS_MANAGEMENT_ACL_ENABLED);
```

Comment:
It's still a bit odd that the `/api/v1/team/visible` endpoint returns a `required` field that indicates to the frontend, in a specific view, whether a team needs to be submitted to the `PUT /api/v1/project` endpoint. `required` doesn't make sense for any other use case. Do you see a way to push this logic to the frontend instead? Then we could just return an array of `Team` here, which from an API standpoint would make a lot more sense.

Reply:
Maybe make the ACCESS_MANAGEMENT_ACL_ENABLED public to view, with the /configProperty/public Endpoint to get from.

Reply:
That sounds reasonable to me. That flag is not confidential so exposing it should be fine.